app/javascript/app/components/my-climate-watch/viz-creator/components/steps/step-four.jsx

```diff
@@ -85,10 +84,10 @@ class Step4 extends Component {
         <div className={styles.saveContainer}>
           {id && (
```

Comment:
This is why we don't have it in the Create modal, isn't it? It would be nice to have always cancel and save